### PR TITLE
Update documentation links

### PR DESCRIPTION
**Pull request by cweiske**
Resolves: https://trac.simpleid.koinic.net/ticket/163

### CODE REVIEW

File: identities/example.user.yml.dist [hunk 1]
```diff
@@ -106,7 +106,7 @@ cert:
     # These two values are then joined together using a semicolon.
     #
     # This option is for advanced users only.  Please see the documentation at
-    # http://simpleid.koinic.net/documentation/advanced-topics/logging-using-client-ssl-certificates
+    # http://simpleid.koinic.net/docs/1/certauth/
```
Comment (kelvinmo) [inline identities/example.user.yml.dist]: Please change URL to http://simpleid.koinic.net/docs/2/auth-schemes/#certauth

File: identities/example.identity.dist [hunk 2]
```diff
@@ -111,7 +111,7 @@ pass="906ae7ffe4d00616daf099c74dcb51b86057f51531123c8039007b53176ae0e5:pbkdf2:sh
 ; These two values are then joined together using a semicolon.
 ;
 ; This option is for advanced users only.  Please see the documentation at
-; http://simpleid.koinic.net/documentation/advanced-topics/logging-using-client-ssl-certificates
+; http://simpleid.koinic.net/docs/1/certauth/
```
Comment (kelvinmo) [inline identities/example.identity.dist]: Please delete this file from the request as this file is no longer used

File: www/core/Protocols/OpenID/OpenIDModule.php [hunk 3]
```diff
@@ -1045,7 +1045,7 @@ public function profileBlocksHook() {
             'id' => 'discovery',
             'title' => $this->t('OpenID 2'),
             'content' => $tpl->render('openid_profile.html', false, $hive),
-            'links' => array(array('href' => 'http://simpleid.koinic.net/documentation/getting-started/setting-identity/claim-your-identifier', 'name' => $this->t('More information'))),
+            'links' => array(array('href' => 'http://simpleid.koinic.net/docs/1/identity-claim/', 'name' => $this->t('More information'))),
```
Comment (kelvinmo) [inline www/core/Protocols/OpenID/OpenIDModule.php]: Please change URL to http://simpleid.koinic.net/docs/2/openid/#claiming

**Comment by cweiske** [comment]
Fixed in https://github.com/simpleid/simpleid/commit/8b36f43528cf14bb26380e005baa7b578f4eb73b

**Pull request closed without merging**